File: mediaufersa.c
#include <stdio.h>
#include <locale.h> 

    int main(){
        setlocale(LC_ALL,"Portuguese");
        float nota1, nota2, nota3, media, recup;
        printf("Digite o valor da nota da sua 1° avaliação : ");
            scanf("%f", &nota1);
        printf("Digite o valor danota da sua 2° avaliação: ");
            scanf("%f", &nota2);
        printf("Digite o valor danota da sua 3° avaliação: ");
            scanf("%f", &nota3);
            media = (nota1 + nota2 + nota3) /3;
        printf("Sua média neste semestre foi %.1f\n ", media);
 if (media >= 7)
 {
    printf("Parabéns! Você foi aprovado");
 } 
 else 
    if (media < 7)
    {
        printf("Você está na recuperação! ");
        printf("Digite o valor da  nota da sua recuperação: ");
        scanf("%f", &recup);
        media = (media + recup) / 2;
            printf("Sua média após a recuperação é %.1f\n ", media);
            if (media >= 5)
            {
               printf("Parabéns! Você passou na recuperação ");
            } else
                printf("REPROVADO! Estude mais na próxima vez! ");
    }     
    return 0;   
}

File: mediaufersa.js
const prompt = require('prompt-sync')();

let nota1, nota2, nota3, media, recup;

    nota1 = Number(prompt("Digite o valor da 1° avaliação:"));
    nota2 = Number(prompt("Digite o valor da 2° avaliação:"));
    nota3 = Number(prompt("Digite o valor da 3° avaliação:"));

    media = (nota1+nota2+nota3)/3;
    media = media.toFixed(1);
    console.log(`Sua média este semestre foi ${media}`)
        if(media >= 7){
            console.log("Parabés! Você foi aprovado")
        } else if(media < 7){
            console.log("Você está na recupeeração!")
            recup = Number(prompt("Digite o valor da nota sua recuperação: "));
                media = (media+recup)/2;
            console.log(`Sua média após a recuperação é ${media}`)
            if(media >= 5){
                console.log("Prabés! Você passou na recuperação e está aprovado")
            } else 
            console.log("REPROVADO! Estude mais na próxima vez")
        }
